lib/errors.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const getContent = (err, debug, translate) => {
+  var content = {};
```

Comment:
Arguably, one should avoid mutation in functional code but I don't think that level of discipline is normal in JavaScript.